articles/azure-monitor/app/opencensus-python-request.md

```diff
@@ -41,6 +42,8 @@ First, instrument your Python application with latest [OpenCensus Python SDK](./
     }
     ```
 
+You can find an example Django application in the sample Azure Monitor OpenCenses Python git repository located [here](https://github.com/Azure-Samples/azure-monitor-opencensus-python/tree/master/docs_samples).
+
 ## Tracking Flask applications
```

Comment:
This document structure of overview, dependencies, requests might be unintuitive for users (since we will be linking to samples that are not explicit separated into overview, dependencies, requests. I think for now simply linking to appropriate pages would be fine but we may want to revisit the structure of our docs. @AaronMaxwell